Blockzilla.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -37,6 +37,15 @@
           "version": "7.9.0"
         }
       },
+      {
```

Comment:
@razvanlitianu is it right that this PR adds this package?

Reply:
Sometimes it removes it, and sometimes it adds it. I'm not entirely sure why but I'm thinking of removing it from the Packages.

Reply:
I will improve the automation so that we don't take this file into account... I will file an issue for that